tests/scripts/generate_test_keys.py

```diff
@@ -100,10 +100,6 @@ def main() -> None:
     args = argparser.parse_args()
 
     output_file = args.output
-    # If the output file already exists, then we can quit (successfully)
-    if os.path.exists(output_file):
-        return
-
     output_file = open(output_file, 'at')
```

Comment:
Pre-existing: this opens the file [in append mode](https://docs.python.org/3/library/functions.html#open). Seems to me we want `'wt'` instead in order to truncate it if it exists.

Reply:
Besides this was also causing the failure on "check_generated_files". Thanks for catching this :)